README.md

```diff
@@ -375,7 +376,11 @@ i18n.runWithLocale(locale, func)
 // keep in mind that locale must be loaded first (if it is not bundled)
 ```
 
-## Integration with React
+## Integrations
+
+This section showcases some of the ways of integrating `universe:i18n` with different frameworks. More detailed examples can be found in the `integrations` directory.
```

Comment:
It'd be awesome to link the `integration` directory (relative path would be enough).